recipes/ocetrac/.ipynb_checkpoints/meta-checkpoint.yaml

```diff
@@ -0,0 +1,49 @@
+# Jinja variables help maintain the recipe as you'll update the version only here.
```

Comment:
I believe this checkpoint file should not be in the PR.

Reply:
Done, thank you.